src/StorageSync/StorageSync/help/Unregister-AzStorageSyncServer.md

```diff
@@ -0,0 +1,236 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/unregister-Azstoragesyncserver
+schema: 2.0.0
+---
+
+# Unregister-AzStorageSyncServer
+
+## SYNOPSIS
+This command will use to unregister storage sync server.
+
+## SYNTAX
+
+### ObjectParameterSet (Default)
+```
+Unregister-AzStorageSyncServer [-Force] [-PassThru] [-AsJob] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
Is this parameter set correct?  Are you allowing unregistering all storage sync services, or is this an error in parameter attributes?